GCDWebServer/Core/GCDWebServerPrivate.h

```diff
@@ -92,6 +92,13 @@
 
 #define __GCDWEBSERVER_LOGGING_FACILITY_COCOALUMBERJACK__
 
+/**
+ * Whether async should be used by log messages, excluding error messages that are always sent sync.
```

Comment:
Please remove this comment which is not needed inside gcdwebserver’s private header.